PyTorchEx1.ipynb

```diff
@@ -0,0 +1,478 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    df = getDF('/nvme/1/ssayyah/nv-wip/amazon_bookreview.json.gz')

</pre><p>Todo Figure out if we can get a download link for this. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>